Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -20,8 +19,14 @@ sub _releases {
 	my $tags       = 'lWRSw';
 	my $library_id = $args->{'library_id'} || $pt->{'library_id'};
 	my $orderBy    = $args->{'orderBy'} || $pt->{'orderBy'};
+	my $menuMode   = $args->{'params'}->{'menu_mode'};
+	my $menuRoles  = $args->{'params'}->{'menu_roles'};
 
-	my %primaryArtistIds = map { Slim::Schema::Contributor->typeToRole($_) => 1 } split(/,/, PRIMARY_ARTIST_ROLES);
+	my $primaryArtistRoles = 'ALBUMARTIST,ARTIST';
+	# add composer to primary artist roles if the user preference requires it.
+	$primaryArtistRoles .= ',COMPOSER' if $prefs->get('useComposerAsArtist')==1;
```

Comment:
We already have `composerInArtists` - `useComposerAsArtist` sounds like a dupe to me. Why should we have two almost identical prefs doing almost identical things ?